pkg/util/namespace/namespace.go

```diff
@@ -7,11 +7,19 @@ import (
 	"strings"
 )
 
-// IsOpenShift returns true if ns is an openshift managed namespace.
+// IsOpenShift returns true if ns is an openshift managed namespace, including the default namespace
 func IsOpenShift(ns string) bool {
 	return ns == "" ||
 		ns == "default" ||
 		ns == "openshift" ||
 		strings.HasPrefix(ns, "kube-") ||
 		strings.HasPrefix(ns, "openshift-")
 }
+
+// IsOpenShiftSystemNamespace returns true if ns is an openshift managed namespace, without the default namespace
+func IsOpenShiftSystemNamespace(ns string) bool {
+	return ns == "" ||
+		ns == "openshift" ||
+		strings.HasPrefix(ns, "kube-") ||
```

Comment:
This would not be enough. We observe customers installing marketplace stuff e.g. `openshift-gitops`. 

At this moment we do not have better solution, so I am just point that for a long term goal we need to either label namespaces we care about or create denylist for namespaces we do not want.

For the sake of this PR I am happy with this solution.

Reply:
We can add additional filtering on the Geneva side if required, and update this list when see if we get too much noise for customer installed components.

Reply:
@petrkotas @UlrichSchlueter I believe @drewandersonnz is going to create a work item for us to loo into filtering out things from customer namespaces which start with `openshift-`, but belong to customers. And @hawkowl also looked into it IIRC.

But yes: this is out of scope of this work item, IMO.